resnet_aws_dist/speedup_resnet_32_2_2_2_factor.txt

```diff
@@ -0,0 +1 @@
+[2.0]
```

Comment:
you need to have the same file under resnet_aws_dist/latest_kpis/ dir to make CE work when PR is merged. see #36 for more details